compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/symbols/TupleTypeSymbol.java

```diff
@@ -33,6 +33,13 @@ public interface TupleTypeSymbol extends TypeSymbol {
      */
     List<TypeSymbol> memberTypeDescriptors();
 
+    /**
+     * Get the tuple members.
+     *
+     * @return {@link List} of member types
+     */
+    List<TupleMemberSymbol> tupleMembers();
```

Comment:
Isn't it more consistent to call this
```suggestion
    List<TupleMemberSymbol> members();
```

Reply:
Since for types we have `memberTypeDescriptors`.

Reply:
+1